ouroboros-consensus-cardano/src/Ouroboros/Consensus/Cardano/CanHardFork.hs

```diff
@@ -270,41 +278,56 @@ type CardanoHardForkConstraints c =
 
 instance CardanoHardForkConstraints c => CanHardFork (CardanoEras c) where
   hardForkEraTranslation = EraTranslation {
-      translateLedgerState   =
-          PCons translateLedgerStateByronToShelleyWrapper
-        $ PCons translateLedgerStateShelleyToAllegraWrapper
-        $ PCons translateLedgerStateAllegraToMaryWrapper
-        $ PCons translateLedgerStateMaryToAlonzoWrapper
-        $ PNil
-    , translateChainDepState =
-          PCons translateChainDepStateByronToShelleyWrapper
-        $ PCons translateChainDepStateAcrossShelley
-        $ PCons translateChainDepStateAcrossShelley
-        $ PCons translateChainDepStateAcrossShelley
-        $ PNil
-    , translateLedgerView    =
-          PCons translateLedgerViewByronToShelleyWrapper
-        $ PCons translateLedgerViewAcrossShelley
-        $ PCons translateLedgerViewAcrossShelley
-        $ PCons translateLedgerViewAcrossShelley
-        $ PNil
-    }
+          translateLedgerState   =
```

Comment:
Style: Please don't increase the indent by an additional four spaces

Reply:
Hmm, not sure how that happened